.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,55 @@
+name: Bug Report
+description: Report a bug encountered with the Kubernetes website
+labels: kind/bug
+body:
+  - type: textarea
+    id: urls
+    attributes:
+      label: URLs
+      value: |
+        URL(s) of the web pages that you were looking at, or were trying to look at.
+      placeholder:
+        https://web-page-url.example/
+    validations:
+      required: true
+
+  - type: textarea
+    id: problem
+    attributes:
+      label: Problem
+      description: |
+        Please provide as much info as possible. The more you can tell us about the bug, the better.
+
+        If the page is not in English, you should either report an issue with the original (English) page
+        instead, or submit a request to get the localized page updated to match the English original.
+
+        If this report is security related, please disclose it privately via https://kubernetes.io/security
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected
+    attributes:
+      label: What did you expect to happen?
```

Comment:
I think contributors are possibly describe redundant content in this section since contributor will describe the problem above section.
If the problem is well described, the expected result of the problem is just result without the problem. (In my experience, usually the answer is "corrected the problem")

Reply:
👍